docs/notebooks/ztf_bts-ngc.ipynb

```diff
@@ -0,0 +1,197 @@
+{
```

Comment:
<p>ZTF is shown with <code>.head()</code>,but you're just showing all of the NGC that you've downloaded. Can we consistently use <code>head()</code>?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>